src/__tests__/Polymesh.ts

```diff
@@ -673,22 +673,13 @@ describe('Polymesh Class', () => {
 
       const params = { did: 'testDid' };
 
-      const result = await polymesh.getIdentity(params);
       const context = dsMockUtils.getContextInstance();
+      const identity = new Identity(params, context);
+      context.getIdentity.onFirstCall().resolves(identity);
 
-      expect(result).toMatchObject(new Identity(params, context));
-    });
-
-    test('should throw an error if the Identity does not exist', async () => {
```

Comment:
Took this case since its covered in the real implementation which is now called.